src/codeu/chat/server/Controller.java

```diff
@@ -151,25 +148,24 @@ private Uuid createId() {
 
     Uuid candidate;
 
-    for (candidate = uuidGenerator.make();
-         isIdInUse(candidate);
-         candidate = uuidGenerator.make()) {
+    for (candidate = uuidGenerator.make(); isIdInUse(candidate); candidate = uuidGenerator.make()) {
```

Comment:
This probably wasn't your original code but just mentioning that a do/while loop would work much better here.